README.md

```diff
@@ -57,7 +57,30 @@ You can then use our tracing dashboards to visualize the requests as traces, whi
 
 If your application uses an OpenTelemetry SDK, you can configure the application to send metrics data to Tanzu Observability using the OpenTelemetry Collector and the Wavefront proxy. Metrics data includes time series, counters, and histograms. When the data is in Tanzu Observability, you can use charts and dashboards to visualize the data and create alerts.
 
-Here's how it works:
+### Directly Send Data Using the Wavefront Proxy - (Recommended)
+<img src="images/opentelemetry_proxy_tracing.png" alt="A data flow diagram that shows how the data flows from your application to the proxy, and then to Tanzu Observability" style="width:750px;"/>
+
+Follow these steps:
+
+1. [Install the Wavefront Proxy](https://docs.wavefront.com/proxies_installing.html) version 12 or higher.
```

Comment:
We still don't know for sure if its version 12 as it hasn't been officially released yet.